<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoViewer</class>
 <widget class="QDialog" name="InfoViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>821</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Information</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="crystalTab">
      <attribute name="title">
       <string>Crystal</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="CrystalInfoDocument" name="crystalInfoDocument" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="atomsTab">
      <attribute name="title">
       <string>Atoms</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="AtomInfoDocument" name="atomInfoDocument" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="surfaceTab">
      <attribute name="title">
       <string>Surface</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="SurfaceInfoDocument" name="surfaceInfoDocument" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="energiesTab">
      <attribute name="title">
       <string>Energies</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="CollapsibleGroupBox" name="groupBox">
         <property name="title">
          <string>Energy Display Options</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="streamlinedLayout">
          <item>
           <widget class="QLabel" name="energyPrecisionLabel">
            <property name="text">
             <string>Energy:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="energyPrecisionSpinBox">
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="spacer1">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="distancePrecisionLabel">
            <property name="text">
             <string>Distance:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="distancePrecisionSpinBox">
            <property name="value">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="spacer2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="energyColorLabel">
            <property name="text">
             <string>Color scheme:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="energyColorComboBox"/>
          </item>
          <item>
           <spacer name="spacer3">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="InteractionInfoDocument" name="interactionsInfoDocument" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>600</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="elasticTensorTab">
      <attribute name="title">
       <string>Elastic Tensor</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="ElasticTensorInfoDocument" name="elasticTensorInfoDocument" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AtomInfoDocument</class>
   <extends>QWidget</extends>
   <header>atominfodocument.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CrystalInfoDocument</class>
   <extends>QWidget</extends>
   <header>crystalinfodocument.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InteractionInfoDocument</class>
   <extends>QWidget</extends>
   <header>interactioninfodocument.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SurfaceInfoDocument</class>
   <extends>QWidget</extends>
   <header>surfaceinfodocument.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ElasticTensorInfoDocument</class>
   <extends>QWidget</extends>
   <header>elastictensorinfodocument.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>collapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
